src/elements/Divider/Divider.tsx

```diff
@@ -23,12 +23,31 @@ export type DividerProps = {
     color?: string;
 };
 
+const styleMap = {
+    [DividerStyle.NoLine]: "tw-border-none",
+    [DividerStyle.Dashed]: "tw-border-dashed",
+    [DividerStyle.Solid]: "tw-border-solid",
+    [DividerStyle.Dotted]: "tw-border-dotted",
+};
+
+const heightMap = {
+    [DividerHeight.Height10]: "tw-h-[10px]",
+    [DividerHeight.Height25]: "tw-h-[25px]",
+    [DividerHeight.Height50]: "tw-h-[50px]",
+    [DividerHeight.Height75]: "tw-h-[75px]",
+    [DividerHeight.Height100]: "tw-h-[100px]",
+};
+
 export const Divider: FC<DividerProps> = ({
     style = DividerStyle.Solid,
     height = DividerHeight.Height50,
     color: borderTopColor = "#CCC",
 }) => (
-    <div className={`tw-flex tw-items-center ${height}`} data-test-id="divider">
-        <hr className={`tw-border-t tw-m-0 tw-w-full ${style}`} style={{ borderTopColor }} data-test-id="divider-hr" />
+    <div className={`tw-flex tw-items-center ${heightMap[height]}`} data-test-id="divider">
```

Comment:
you could use the `merge` function for consistency but i'm also fine with the current solution 😉